client/src/pages/profile/index.tsx

```diff
@@ -208,13 +210,20 @@ export class ProfilePage extends React.Component<Props, State> {
     try {
       const githubId = router.query ? (router.query.githubId as string) : undefined;
       const profile = await this.userService.getProfileInfo(githubId);
+      let isProfileOwner;
```

Comment:
for now, as we don't have any additional requirements, I would do it simple true/false, without undefined.

```
         const userId = this.props.session.githubId;
         const profileId = profile.generalInfo!.githubId;
         const  isProfileOwner = checkIsProfileOwner(userId, profileId);
```